src/openms/include/OpenMS/ANALYSIS/ID/PeptideSearchEngineAlgorithm.h

```diff
@@ -44,7 +44,7 @@ class OPENMS_DLLAPI PeptideSearchEngineAlgorithm :
   protected:
     void updateMembers_() override;
 
-    FragmentIndexScorer fragment_index_;
+    //FragmentIndex fragment_index_;
```

Comment:
```suggestion
```

Reply:
note: if you want to accept suggestions don't click on "Commit suggestion" but "Add suggestion to batch". Otherwise we will run our CI for every suggested commit.